styles/styles.css

```diff
@@ -416,6 +435,93 @@ nav ul li:hover {
   background-size: contain;
 }
 
+.popup {
+  width: 90%;
+  height: 90%;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  overflow: hidden;
+  padding: 1rem;
+}
+
+#popup-title {
+  margin-bottom: 40px;
+  position: relative;
+}
+
```

Comment:
- Good work implementing the popup window. Kindly check your images and icons on projects, they are not visible

![image](https://user-images.githubusercontent.com/93630700/223745134-b59f4ee5-b72c-48d8-b082-2cb1a2e185d9.png)